src/quantuminspire/projectq/backend_qx.py

```diff
@@ -74,22 +74,28 @@ def __init__(self, num_runs: int = 1024, verbose: int = 0, quantum_inspire_api:
                 raise AuthenticationError('Make sure you have saved your token credentials on disk or '
                                           'provide a QuantumInspireAPI instance as parameter to QIBackend') from ex
         self.quantum_inspire_api: QuantumInspireAPI = quantum_inspire_api
-        self.backend_type: Optional[Union[Dict[str, Any], int, str]] = backend_type
+        self.backend_type: Optional[Union[int, str]] = backend_type
+        backend = self.quantum_inspire_api.get_backend_type(backend_type)
+        self.is_simulation_backend = not backend["is_hardware_backend"]
```

Comment:
is there a reason for exposing this one?

Reply:
nope, I also checked the rest of the members